src/pages/index.js

```diff
@@ -101,7 +225,118 @@ const cardSection = new Section(
 );
 
 //initialize instances
-cardSection.renderItems();
+
+//Initial Cards
+api
+  .getInitialCards()
+  .then((cards) => {
+    cardSection.renderItems(cards);
+  })
+  .catch((error) => {
+    console.error(`Error: ${error}`);
+  });
+
 cardPreviewPopup.setEventListeners();
 profileFormPopup.setEventListeners();
 postFormPopup.setEventListeners();
+profilePictureFormPopup.setEventListeners();
+confirmDeletePopup.setEventListeners();
+
+api
+  .getUserInfo()
+  .then((res) => {
+    return (
+      userInfo.setUserInfo({ name: res.name, job: res.about }),
+      (profilePicture.src = res.avatar)
+    );
+  })
+  .catch((error) => {
+    console.error(`Error: ${error}`);
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors